guidelines/coding-guidelines.md

```diff
@@ -11,3 +11,21 @@ For public facing interfaces we favor read-only collections and enumerables. For
 * Avoid using `System.Linq`
 * Avoid using `foreach` over collections that do not have a struct enumerator
 
+## Don't use string interpolation
+
+By using string interpolation it is impossible for logging frameworks that support structure logging to distinguish values within the string. We must use the normal argument based overload so logging frameworks like Serilog can capture structured data.
+
+Wrong:
+
+```c#
+logger.Info("File share data bus started. Location: " + basePath);"
```

Comment:
In this example here for the wrong, it looks more like you're against string concatenation than interpolation?
The example for right, looks more like interpolation.
Can you explain a bit more about your concern?